bazel/setup_clang.sh

```diff
@@ -14,7 +14,7 @@ export PATH="$(${LLVM_PREFIX}/bin/llvm-config --bindir):${PATH}"
 RT_LIBRARY_PATH="$(dirname $(find $(llvm-config --libdir) -name libclang_rt.ubsan_standalone_cxx-x86_64.a | head -1))"
 
 echo "# Generated file, do not edit. If you want to disable clang, just delete this file.
-build:clang --action_env=PATH=${PATH}
+build:clang --action_env='PATH=${PATH}'
```

Comment:
do you mind fix other env vars below which potentially contain whitespaces?

Reply:
For safety, whole values that passed to bazel should be inside single quotes. I will fix them.